lib/src/metta/runner/stdlib.rs

```diff
@@ -24,6 +24,22 @@ use regex::Regex;
 use super::arithmetics::*;
 use super::string::*;
 
+macro_rules! grounded_op {
+    ($name:ident, $disp:literal) => {
+        impl PartialEq for $name {
+            fn eq(&self, _other: &Self) -> bool {
+                true
+            }
+        }
+
+        impl Display for $name {
```

Comment:
```suggestion
        impl std::fmt::Display for $name {
```

Reply:
Display isn't in the preamble like PartialEq, so if we want to export this macro it will be good to specify it by full path.